### DISCUSSION TITLE
Offer to triage

### DISCUSSION THREAD

**Discussion by ejcheng** [faker-js/faker#149] [General]
Just like @Shinigami92, I would love to triage for Faker.

Background: I've triaged for Express for a decent amount of time now, I would love to bring this experience to Faker and help triage issues.

I just need triage permissions and "green-tick-mark" permissions when reviewing PRs to do this.

Thank you for your consideration!

**Comment by Shinigami92** [comment]
Please jump into our discord https://discord.com/invite/4qDjAmDj4P

**Reply by ejcheng** [reply]
Just jumped into the server

**Reply by Shinigami92** [reply]
What's your name there? Don't see an import-brian ^^

**Reply by ejcheng** [reply]
> What's your name there? Don't see an import-brian ^^

I'm in the server as import-brain#4126, might need to refresh or something

**Reply by Shinigami92** [reply]
You now have triage role in discord
For GitHub stuff, please ask @damienwebdev :slightly_smiling_face: 
Maybe we need to setup a new team that can make just green ticks :slightly_smiling_face:

**Reply by ejcheng** [reply]
> You now have triage role in discord For GitHub stuff, please ask @damienwebdev 🙂 Maybe we need to setup a new team that can make just green ticks 🙂

Yes, I think some sort of team with triage and green tick permissions would be perfect :)
